src/server/game/Entities/Player/Player.cpp

```diff
@@ -2626,6 +2626,7 @@ bool Player::IsGroupVisibleFor(Player const* p) const
         default: return IsInSameGroupWith(p);
         case 1:  return IsInSameRaidWith(p);
         case 2:  return GetTeam() == p->GetTeam();
+        case 3:  return nullptr;
```

Comment:
It doesn't even compile (with a properly configured compiler): https://travis-ci.org/TrinityCore/TrinityCore/builds/188351642#L1047 :)